src/pyuvdata/uvbeam/mwa_beam.py

```diff
@@ -617,12 +617,11 @@ def read_mwa_beam(
         ):
             self.history += self.pyuvdata_version_str
 
-        self.x_orientation = "east"
-
         self._set_efield()
         self.Naxes_vec = 2
         self.Ncomponents_vec = 2
         self.feed_array = np.array([str(pol.lower()) for pol in pol_names])
+        self.feed_angle = np.where(self.feed_array == "x", np.pi / 2, 0.0)
```

Comment:
should we specify a mount type? I'm confused about whether it should be fixed or phased...

Reply:
I think "phased" is the most appropriate here, or at least the one that matches the limited documentation on it UVFITS (an "electronically-steered" primary beam).

Reply:
Added in 4bd22a9260ec0b4845b31f0bc4d84d7f61e23587.